CarAccidentScenario/EmergencyServicesException.java

```diff
@@ -0,0 +1,7 @@
+package CarAccidentScenario;
+
+public class EmergencyServicesException extends Exception {
```

Comment:
This name is not letting me know what the exact problem is, is it that emergency services are unavailable, or that services were denied or unable to contact emergency services or something else.
Please rename it to something more specific.

Reply:
Done